server/FaceEmotionAnalyzerAPI/src/main/java/com/face/response/FaceApiErrorResponse.java

```diff
@@ -0,0 +1,31 @@
+package com.face.response;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@JsonIgnoreProperties(ignoreUnknown = true) 
+public class FaceApiErrorResponse{
+		
+		private String message;
```

Comment:
```suggestion
              @JsonProperty("error")
		private String message;
```
getterとかに `@JsonProperty`がついてますが、↑だと駄目なんですかね？
基本的なこと聞いていたらごめんなさい・・